site/cypress/e2e/Cypress-Feature/forums.spec.js

```diff
@@ -11,6 +11,41 @@ const reply3 = 'Cypress Reply 3 Cypress';
 const merged1 = 'Merged Thread Title: '.concat(title3, '\n\n', content3);
 const merged2 = 'Merged Thread Title: '.concat(title2, '\n\n', content2);
 const attachment1 = 'sea_animals.png';
+const isVisible = 'be.visible';
+const notVisible = 'be.not.visible';
+
+const goToForum = (user) => {
+    cy.login(user);
+    cy.visit(['sample']);
+    cy.get('#nav-sidebar-forum').click();
```

Comment:
No need to add this line, you can directly go to that page

Reply:
I am not sure what this means. Could you explain further?

Reply:
I just realized, fixed in [651b049](https://github.com/Submitty/Submitty/pull/10599/commits/651b049a1e096a74f342e82596504f1c5f0a1bc8)